.github/workflows/sql-test-workflow.yml

```diff
@@ -93,3 +93,5 @@ jobs:
           protocol/build/reports/**
           legacy/build/reports/**
           plugin/build/reports/**
+          doctest/build/testclusters/docTestCluster-0/logs/*
+          integ-test/build/testclusters/*/logs/*
```

Comment:
These are unrelated but useful additions. Correct?

Reply:
Sometimes cluster logs are required to understand the failure.

Reply:
nice addition